datalab/datalab_session/data_operations/stacking.py

```diff
@@ -54,12 +53,11 @@ def operate(self):
             input_fits_list.append(InputDataHandler(input['basename'], input['source']))
             self.set_operation_progress(0.5 * (index / len(input_files)))
 
-        cropped_data = crop_arrays([image.sci_data for image in input_fits_list])
-        stacked_ndarray = np.stack(cropped_data, axis=2)
```

Comment:
Again we can avoid doubling the memory with the stack by just summing the cropped arrays directly